aws/aws-how-to/instances/import-local-vm-to-aws.rst

```diff
@@ -0,0 +1,137 @@
+Importing a local Ubuntu VM to AWS
+==================================
+
+This document will guide you to import local virtual machines to AWS.
+Please refer to the official `AWS Documentation`_ for more details. 
+
+Requirements
+------------
+
+- An AWS account
+- AWS CLI installed and configured
+- A virtual machine exported in Open Virtual Appliance (OVA) format. AWS also supports Virtual Machine Disk (``VMDK``), Virtual Hard Disk (``VHD/VHDX``), and raw.
+
+.. note::
+        Users running Ubuntu on KVM, OpenStack, LXD and other virtualization environments will most likely have ``qcow2`` files and not ``OVA``. Although ``qcow2`` is not supported, it is possible to use ``qemu-img`` to convert it to raw as follows: 
+        
+        ``sudo qemu-img convert -O raw /var/lib/libvirt/images/my-machine.qcow2 my-machine.raw``
+    
+
+
+Upload your OVM machine image to S3
```

Comment:
```suggestion
Upload your VM image to S3
```

Reply:
I think the format is relevant

Reply:
Then can we mention OVA (instead of OVM) and the others too if required.

Reply:
You re absolutely right. It should be OVA.

Reply:
done in the new commit